cmake/EthOptions.cmake

```diff
@@ -11,14 +11,20 @@ macro(configure_project)
 	option(VMTRACE "Enable VM tracing" OFF)
 	option(PROFILING "Enable profiling (deprecated)" OFF)
 	option(FATDB "Enable fat state database" ON)
-	option(ROCKSDB "Build with RocksDB instead of LevelDB" OFF)
 	option(PARANOID "Enable additional checks when validating transactions (deprecated)" OFF)
 	option(MINIUPNPC "Build with UPnP support" OFF)
 	option(FASTCTEST "Enable fast ctest" OFF)
 
+	if(WIN32 OR NOT HUNTER_ENABLED)
+		find_package(LevelDB REQUIRED)
+	else()
+	  hunter_add_package(leveldb)
```

Comment:
We can have this change added, but not in this PR. Please remove it for now.

Reply:
Also, format CMake with indentation of 4 spaces.

Reply:
@chfast I rolled back this change.